hunter_base/hunter_base.xml

```diff
@@ -0,0 +1,9 @@
+<library path="hunter_base">
+  <class name="hunter_base/DiffBotSystemHardware"
+         type="hunter_base::DiffBotSystemHardware"
+         base_class_type="hardware_interface::SystemInterface">
+    <description>
```

Comment:
is hardware_interface declared a run dependency? Overall, please check that all run dependencies are declared.